src/content/en/2021/compression.md

```diff
@@ -41,34 +43,39 @@ HTTP compression is recommended for text-based content, such as [HTML](./markup)
   )
 }}
 
-Compared to the other content types, `text/plain` and `text/html` uses the least amount of compression, with merely 12% and 14% using compression at all. This might be because `text/html` is more often dynamically generated than static content such as JavaScript and CSS. However, it is recommended to compress dynamic content as well, and the next sections show how to configure this.
+Compared to the other content types, `text/plain` and `text/html` uses the least amount of compression, with merely 12% and 14% using compression at all. This might be because `text/html` is more often dynamically generated than static content such as JavaScript and CSS. However, it is also recommended to compress dynamic content. The next sections show how this is configured.
 
 ## Server settings for HTTP compression
 
 For HTTP content encoding, the HTTP standard defines the [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) request header, with which a HTTP client can announce to the server what content encodings it can handle. The server's response can then contain a [Content-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding) header field that specifies which of the encodings was chosen to transform the data in the response body.
 
-Both Brotli and Gzip are supported by virtually all browsers. On the server side, most popular servers like nginx and Apache can be configured to use Brotli and/or Gzip. We want to give some pointers on how to best configure text compression on your server. There are two types of compression settings:
+Both <a hreflang="en" href="https://github.com/google/brotli">Brotli</a> and <a hreflang="en" href="https://www.gnu.org/software/gzip/manual/gzip.html">Gzip</a> are supported by virtually all browsers. On the server side, most popular servers like nginx and Apache can be configured to use Brotli and/or Gzip. There are two types of compression settings:
 
  - for precompressed (static) content: this content is already compressed beforehand, ideally with the highest level possible, and the web server should be set up to find the appropriate compressed files based on the filename extension, for example.
  - for dynamic content, which is compressed on the fly for each request by the web server (or a plugin) itself, for dynamically generated text content
 
-When compressing text with Brotli or Gzip it is possible to select different compression levels. Higher compression levels will result in smaller compressed files, but take a longer time to compress. During decompression, CPU usage tends not to be higher for more heavily compressed files. Rather, files that are compressed with a higher compression level are slightly faster to decode.
+{# TODO: these bullets dont flow well gramatically but i don't understand it enough to fix it #}
```

Comment:
I have rewritten these bullet points, I have also taken the understanding as feedback, and therefore added slightly more context around this, by adding "The configuration is different depending on when the content is generated" to the previous sentence